src/app/sessions/containers/sessions-propositions-search/sessions-propositions-search.component.ts

```diff
@@ -0,0 +1,62 @@
+import { Component, OnInit } from '@angular/core';
+import * as JudgeActions from '../../../judges/actions/judge.action';
+import * as RoomActions from '../../../rooms/actions/room.action';
+import * as SessionActions from '../../../sessions/actions/session.action';
+import { State } from '../../../app.state';
+import { select, Store } from '@ngrx/store';
+import * as fromRooms from '../../../rooms/reducers/room.reducer';
+import { Judge } from '../../../judges/models/judge.model';
+import { map } from 'rxjs/operators';
+import * as fromSessionIndex from '../../reducers';
+import { Room } from '../../../rooms/models/room.model';
+import { Observable } from 'rxjs/Observable';
+import * as fromJudges from '../../../judges/reducers';
+import { SessionPropositionQuery } from '../../models/session-proposition-query.model';
+import { SessionPropositionView } from '../../models/session-proposition-view.model';
+import { combineLatest } from 'rxjs/observable/combineLatest';
+
+@Component({
+  selector: 'app-sessions-propositions-search',
+  templateUrl: './sessions-propositions-search.component.html',
+  styleUrls: ['./sessions-propositions-search.component.scss']
+})
+export class SessionsPropositionsSearchComponent implements OnInit {
+
+  judges$: Observable<Judge[]>;
+  rooms$: Observable<Room[]>;
+  sessionPropositions$: Observable<any>;
+  judgesLoading$: Observable<boolean>;
+  roomsLoading$: Observable<boolean>;
+  filterDataLoading$: Observable<boolean | false>;
+  sessionPropositionsLoading$: Observable<boolean | false>;
+
+  constructor(private store: Store<State>) {
+    this.rooms$ = this.store.pipe(select(fromSessionIndex.getRooms), map(this.asArray)) as Observable<Room[]>;
+    this.judges$ = this.store.pipe(select(fromJudges.getJudges), map(this.asArray)) as Observable<Judge[]>;
+    this.sessionPropositions$ = this.store.pipe(select(fromSessionIndex.getFullSessionPropositions), map(this.asArray));
+    this.sessionPropositionsLoading$ = this.store.pipe(select(fromSessionIndex.getSessionsLoading));
+    this.roomsLoading$ = this.store.pipe(select(fromRooms.getLoading));
+    this.judgesLoading$ = this.store.pipe(select(fromJudges.getJudgesLoading));
+    this.filterDataLoading$ = combineLatest(this.roomsLoading$, this.judgesLoading$
```

Comment:
I can't see this.roomsLoading$ and this.judgesLoading$ used anywhere despite that 'combineLatest' so no need to keep them as variables. You could do:

combineLatest(this.store.pipe(select(fromRooms.getLoading)), this.store.pipe(select(fromJudges.getJudgesLoading)), (...))